python/python/glide/async_commands/core.py

```diff
@@ -1415,6 +1415,33 @@ async def lindex(
             await self._execute_command(RequestType.LIndex, [key, str(index)]),
         )
 
+    async def lset(self, key: str, index: int, element: str) -> TOK:
```

Comment:
Some commands return `TOK`, some - `str`, even though they return only `OK`.
What is the client standard?

Reply:
TOK is just a python way for making the literal OK a type, it's the same as str but for commands that only returns OK